bundles/mapping/drawtools/plugin/DrawPlugin.ol3.js

```diff
@@ -405,7 +412,13 @@ Oskari.clazz.define(
          */
         getFeatures: function (layerId) {
             var me = this,
-                features = [];
+                features = [],
+                layer = me.getLayer(layerId);
+
+            if(!layer) {
+                return features;

```

Comment:
If no layer with `layerId` is found, it would be better to return `null` to distinguish from the case where the layer was found but it had zero features. Other code calling this method would need to be updated to handle the case, of course.